### DISCUSSION TITLE
Atomic Calendar does not load on Samsung FrameTV

### DISCUSSION THREAD

**Discussion by daj102** [totaldebug/atomic-calendar-revive#1408] [Support] [answered]
I have a 2022 Samsung FrameTV, which runs Tizen.  Although it has an 'app store' for some things, for internet there is only the built-in browser.  I've been using that to display a 'dashboard' generated in MagicMirror, but I realized most of the things I display are coming from HomeAssistant anyway, and I want to display a Lovelace view instead.

Most of the cards I have in the view (Mushroom entities and buttons, a couple of weather cards, all inside a layout-card) work fine ... but the Atomic calendar card does not display.  I get the translucent red box with 'card not loaded' error, and a dump of the JSON config.  I know that the component is installed correctly - I run the same view in my browser and on a Google Hub Max with no problems.  It must be an incompatibility with the Tizen browser ... but I have no idea how to try and work out what the issue is.  It being a TV, there aren't a lot of dev tools to use ... or even documentation.  The most expansive description of what the browser can/cannot support is on [a third-party website](https://www.lifewire.com/samsung-smart-tv-web-browser-4686055), although nothing there looks as if would affect a simple calendar card.

I'm going to try and strip down the component to virtually nothing and build it back up to see if I can find that 'one thing' that breaks it, but I suspect it's a component-level thing.  Does anyone have any ideas how I can go about debugging this and ... hopefully ... fixing it?

**Comment by marksie1988** [comment, marked as answer]
Try install v9.1.0 or v9.0.0, if the card works on that version it is because of a recent change made by dependant plugins to drop support for older browsers.

**Reply by daj102** [reply]
Hmmmm ... neither of those are available via HACS any more (v9.2.1 and above), so I shutdown HASS, and replaced the contents of the atomic-calendar-revive.js file with v9.0.2 that I downloaded from GitHub.  The Tizen browser doesn't have an incognito mode, so I did delete the browsing history, cookies, etc as much as I could.

v9.0.2 seems to work; I confirmed it was using that version by truncating the title to 3 characters (you added the ... suffix in v9.5.0).  You mentioned 'dependent plugins' ... so am I stuck on older versions of ACR, or is there a way to retrofit those plugins?

**Reply by marksie1988** [reply]
I may be able to create a release for older versions but not sure yet how to do that.

**Reply by daj102** [reply]
Yeah, fair enough - totally understand.  For now, it's working, and with card-mod I've been able to style it a bit so it looks very similar to the MagicMirror calendar I use (although you might want to look at that module ~to rip off some ideas~ for inspiration ;) ).  What I might try and do is fork an older version and edit it a bit to create a 'FrameTV' version of the component for just that view, while the rest of my views use the latest/greatest.

Thanks for all your effort - it's a great component.

**Reply by marksie1988** [reply]
I think I can create a different package file and do 2 releases whenever I update
